pkg/config/config.go

```diff
@@ -99,6 +101,12 @@ func (cfg *Config) BindFlags() {
 		[]string{},
 		"Configures the ACK service controller to always set key/value pairs tags on resources that it manages.",
 	)
+	flag.StringVar(
+		&cfg.WatchNamespace , flagWatchNamespace,
+		"",
+		"Specific namespace the service controller will watch for object creation from CRD. "+ 
```

Comment:
Change the above to read:

"Zero or more Kubernetes namespaces the controller will watch for changes in."

The reason is because this change does *not* make the controller only watch for *objects* being created in specific namespaces. What this change does is make the controller only watch for specific *namespaces* having their annotations changed. There's a big difference between the two things.

Reply:
Ignore the above.